tests/topotests/ospf_sr_te_topo1/test_ospf_sr_te_topo1.py

```diff
@@ -382,12 +382,12 @@ def check_bsid(rt, bsid, fn_name, positive):
     candidate_output = ""
     # First wait for convergence
     tgen = get_topogen()
-    while count < 30:
+    router = tgen.gears[rt]
+    while count < 180:
```

Comment:
This def has sleep(1) increasing this to 180 will add up another 150 sec extra in worst case. Better if we fix that static sleep also.

Reply:
this was a test to see what the hell is going on and if it's merely a timeout situation that needed longer.

finally I really don't understand how you `fix` the static sleep here.  This step is waiting for convergence.  You always have to sleep somewhere in a test system.  I just want the sleeps to be focused on dynamically waiting for some convergence event instead of sleeping and then moving on to another step in the test without testing that you have converged

Reply:
One way i can think of is instead of sleep(1), we can run router.vtysh_cmd("show mpls table json") multiple times. So that we can avoid static sleep.